Formula/terminator.rb

```diff
@@ -0,0 +1,37 @@
+class Terminator < Formula
+  include Language::Python::Virtualenv
+
+  desc "Multiple GNOME terminals in one window"
+  homepage "https://gnome-terminator.org"
+  url "https://github.com/gnome-terminator/terminator/archive/refs/tags/v2.1.1.tar.gz"
+  sha256 "ee1907bc9bfe03244f6d8074b214ef1638a964b38e21ca2ad4cca993d0c1d31e"
+  license "GPL-2.0-only"
+
+  depends_on "pygobject3"
+  depends_on "python@3.9"
+  depends_on "six"
+  depends_on "vte3"
+
+  resource "psutil" do
+    url "https://files.pythonhosted.org/packages/e1/b0/7276de53321c12981717490516b7e612364f2cb372ee8901bd4a66a000d7/psutil-5.8.0.tar.gz"
+    sha256 "0c9ccb99ab76025f2f0bbecf341d4656e9c1351db8cc8a03ccd62e318ab4b5c6"
+  end
+
+  resource "configobj" do
+    url "https://files.pythonhosted.org/packages/64/61/079eb60459c44929e684fa7d9e2fdca403f67d64dd9dbac27296be2e0fab/configobj-5.0.6.tar.gz"
+    sha256 "a2f5650770e1c87fb335af19a9b7eb73fc05ccf22144eb68db7d00cd2bcb0902"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    pid = Process.spawn bin/"terminator", "-d", [:out, :err] => testpath/"terminator.out"
+    sleep 5
+    Process.kill "TERM", pid
+    assert_match "Window::create_layout: Making a child of type: Terminal", (testpath/"terminator.out").read
+  ensure
+    Process.kill "KILL", pid
+  end
```

Comment:
Hmm, the test's not working. I suspect it's happening here. Let's get rid of it.
```suggestion
```
My bad for suggesting this :) Please squash all your commits together when you make this change.

Reply:
Oh, geez, that was not actually was causing the problem. 

I didn't realized you had made this commit prior so I had to merge your changes and mine when I pushed,

Reply:
I didn't actually push anything to your branch, so I'm not sure why you needed to merge anything.

Reply:
Ah, maybe I just got confused then.  I'm squashing the commits now

Reply:
There. Commits all squashed now.

I can't say too many times how impressed I am with all your help. As a guy who maintains an open source project myself I know how hard that can be.  Thanks again, and cheers!

Reply:
No problem! You've done most of the hard work; I'm just here to help you finish it off and suggest some tweaks to help make the formula easier to maintain.

Reply:
Thanks again.  This seems like it's all working with your gobject-introspection fixes now.